### DISCUSSION TITLE
Spatial Movement - Showcase

### DISCUSSION THREAD

**Discussion by avarga** [inet-framework/inet#609] [Showcases]
Discuss the "Spatial Movement" showcase here. Ideas and recommendations are welcome.

**Comment by rajasreeosm** [comment]
Hi, I am new to omnetpp
![Screenshot 2024-11-05 193617](https://github.com/user-attachments/assets/115a61a3-6a2e-4d50-8afd-bd17979d4884)
When i am trying to run this Spatial mobility its showing this error. Could you please guide me on it. Also after simulating this model how will i take the results and export it as csv file

**Comment by avarga** [comment]
The "Visualization OSG (3D)" project feature is not enabled. Enable it in the Project menu, then recompile INET. For it to work, OMNeT++ must also be compiled with OSG support enabled, check Makefile.inc.

**Reply by rajasreeosm** [reply]
Thankyou so much it worked. I am trying to do a sybil attack detection in UAV, so i have done the modifications in the omnetpp.ini file and ned file. I want to include algorithms to detect the sybil nodes using the  RSSD(Received Signal Strength Difference) and TDoA(Time Difference of Arrival). So i am confused about creating the c++ files and including the recording of the signals from the simulation in the algorithm. It would be great if you could provide any assistance to this.
